Documentation/network/servicemesh/ingress.rst

```diff
@@ -154,6 +159,108 @@ using the ``ingressController.ingressLBAnnotationPrefixes`` setting.
+
+The default behavior regarding shared or dedicated ingress can be configured via
+``ingressController.loadbalancerMode``.
+
+.. warning::
+    Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available on all Cilium Nodes where Cilium ingress controller Envoy listeners are exposed.
+
+Bind to privileged port
+***********************
+By default, it's mandatory to use ports higher than ``1023`` as the Cilium L7
+Envoy process does not have any Linux capabilities out-of-the-box (so no
+``CAP_NET_BIND_SERVICE``) and is therefore not allowed to listen on privileged
+ports.
+If you choose a port equal to or lower than ``1023``, ensure that
+``envoy.securityContext.capabilities.keepCapNetBindService=true`` is configured
+and to add the capability ``NET_BIND_SERVICE`` to the respective
+:ref:`Cilium Envoy container via Helm values<envoy>`:
+
+* Standalone DaemonSet mode: ``envoy.securityContext.capabilities.envoy``
+* Embedded mode: ``securityContext.capabilities.ciliumAgent``
+
+Configure the following to allow privileged port bindings in host network mode:
+
+.. code-block:: yaml
+
+    ingressController:
+      enabled: true
+      hostNetwork:
+        enabled: true
+    # For Cilium Envoy standalone DaemonSet mode:
+    envoy:
+      enabled: true
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+          envoy:
+          # Add NET_BIND_SERVICE to the list (keep the others!)
+          - NET_BIND_SERVICE
+    # For Cilium Envoy embedded mode:
+    envoy:
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+    securityContext:
+      capabilities:
+        ciliumAgent:
+        # Add NET_BIND_SERVICE to the list (keep the others!)
+        - NET_BIND_SERVICE
+
+Deploy Gateway API listeners on subset of nodes
+***********************************************
+It's supported to expose the Cilium ingress controller Envoy
+listener only on specific nodes. This only works in combination with the
+host network mode and can be configured via a node label selector in the Helm
+values.
+
+.. code-block:: yaml
+
+    ingressController:
+      enabled: true
+      hostNetwork:
+        enabled: true
+        nodes:
+          matchLabels:
+            role: infra
+            component: ingress
+
+An empty selector selects all nodes and continues to expose the functionality on
```

Comment:
put the following sentence in front - as we do for Gateway API

> This will deploy the Ingress Controller Envoy listener only on the Cilium Nodes matching the configured labels.